### PR TITLE
Publish to live 2020-09-23

### PR DESCRIPTION
**Pull request by smitpatel**

### CODE REVIEW

File: entity-framework/core/miscellaneous/context-pooling.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: DbContext Pooling
+description: DbContext pooling in Entity Framework Core
+author: rick-anderson
+ms.author: riande
+ms.date: 9/19/2020
+uid: core/miscellaneous/dbcontextpool
```
Comment (smitpatel) [inline entity-framework/core/miscellaneous/context-pooling.md:7]: file name should match the uid for consistency.

File: entity-framework/core/miscellaneous/context-pooling.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Apps should be profiled and tested to show that context initialization is a significant cost.
+
+`AddDbContextPool` has a few limitations on what can be done in the `OnConfiguring` method of the context.
+
+> [!WARNING]  
+> Avoid using context pooling in apps that maintain state. For example, private fields in the context that shouldn't be shared across requests. EF Core only resets the state that it is aware of before adding a context instance to the pool.
+
+Context pooling works by reusing the same context instance across requests. This means that it's effectively registered as a [Singleton](/aspnet/core/fundamentals/dependency-injection#service-lifetimes) in terms of the instance itself so that it's able to persist.
+<!-- Review, able to persist what?  -->
```
Comment (smitpatel) [inline entity-framework/core/miscellaneous/context-pooling.md:38]: pending review.

File: entity-framework/toc.yml [hunk 3]
```diff
@@ -78,6 +78,8 @@
         href: core/miscellaneous/async.md
       - name: Logging
         href: core/miscellaneous/logging.md
+      - name: Connection pooling
```
Comment (smitpatel) [inline entity-framework/toc.yml:81]: wrong title.

**Comment by smitpatel** [comment]
@roji @Rick-Anderson - Please fix issues in review comment above by submitting a PR against master.

**Comment by Rick-Anderson** [comment]
@smitpatel  I'll fix it. @roji  suggested changing the file name and I forgot about uid. Some folks consider the `uid` the PK.

**Comment by smitpatel** [comment]
Yes, uid is like PK for us also. We use it for cross-ref inside this docset. Just to avoid breaking things unintentionally when renaming file, we just keep it same as the file name so we know when we are making breaking changes.

**Pull request merged**